routes/authRouter.js

```diff
@@ -0,0 +1,29 @@
+const Router  = require('koa-router')
+const { login, registration } = require("../controllers/authController");
+const {tokenGenerator} = require("../helper/tokenGenerator");
+const {authMiddleware} = require("../middlewares/authMiddleware");
+
+const router = new Router()
+
+
+router.post('/login', async (ctx) => {
+    const userData = ctx.request.body
+
+    ctx.body = await login(userData)
```

Comment:
As I said in the last pull request, don't use `ctx.body  = ` in all routers, instead create a middleware and do it in 1 place.
Your router can return the value, but not assign anything to `ctx.body`

Reply:
Done